pcdsdevices/timetool.py

```diff
@@ -11,19 +11,22 @@
 from .inout import CombinedInOutRecordPositioner
 
 
-class TimeTool(CombinedInOutRecordPositioner):
+class Timetool(CombinedInOutRecordPositioner):
```

Comment:
Yeah...
There was some conflict between it being called `Timetool` and `TimeTool` and since it hasn't been used anywhere yet I thought it wouldn't matter which I picked and `Timetool` looks nicer so I went with it. I guess I should've chosen the class's version just to be safe :/

Reply:
If it's not used here or in happi or in a beamline profile, I suppose it's fine (@ZLLentz?)

Reply:
It's still a new enough class that it should be ok